setup.py

```diff
@@ -7,7 +7,7 @@
     author='Martin Siggel',
     author_email=',martin.siggel@dlr.de',
     license='Apache-2.0',
-    requires=['tigl3', 'tigl', 'tixi', 'semver', 'numpy'],
+    install_requires=['tigl3', 'tigl', 'semver', 'numpy'],
```

Comment:
Why did you remove tixi from the requirements? Was this needed to get the code running? I know it is a transitive dependency of tigl and tigl3, but since it is extensively used directly in cpacs2to3, I think it makes sense to have it as a dependency here from a general code design perspective.